OSDOCS-16945# CQA work Stor1-- non-graceful shutdown (#106592)
openshift/openshift-docs: modules/persistent-storage-csi-vol-detach-non-graceful-shutdown-procedure.adoc

+
[IMPORTANT]
====
Tainting a node this way deletes all pods on that node. This also causes any pods that are backed by statefulsets to be evicted, and replacement pods to be created on a different node.
====
+
[source,terminal]
----
$ oc adm taint node <node_name> node.kubernetes.io/out-of-service=nodeshutdown:NoExecute
----
+

> * `<node_name>` = name of the node that shut down non-gracefully

Reviewer: ```suggestion
Use the `<node_name>` to specify the node that shut down non-gracefully.
```
